packages/react-router-dom/modules/NavLink.js

```diff
@@ -5,68 +5,87 @@ import invariant from "tiny-invariant";
 import Link from "./Link";
 import { resolveToLocation, normalizeToLocation } from "./utils/locationUtils";
 
+// React 15 compat
```

Comment:
Duplicated, makes it easier to just delete this code w/o messing around with files and imports and stuff, fine if somebody wants me to put it into `utils/shimForwardRef.js` or something.

Reply:
I like the duplicated code approach. Makes it really easy to spot later when we need to remove it.